server/controllers/chartsController/helpers.js

```diff
@@ -144,12 +149,13 @@ export const postProcessData = (data, studyTotals) => {
   for (const [key, count] of data) {
     const [study, valueLabel, targetValue] = key.split('-')
     const totalsForStudy = studyTotals[study]
-    const totals = totalsForStudy.targetTotal || totalsForStudy.count
+    const totals = totalsForStudy?.targetTotal || totalsForStudy?.count
```

Comment:
what is the reason for this change? When would we not have totals for a study or counts for a study?

Reply:
when we have a new site with new data these are some of the scenarios. The chart form hasn't been edited to add a target so we don't have a way to calculate it and if the new data does not have a value for a variable